packages/terra-status-view/src/StatusView.jsx

```diff
@@ -98,29 +98,43 @@ class StatusView extends React.Component {
    * with 40% of the remaining space at the top with 60% on the bottom.
    */
   showAndCenterItems() {
-    const viewHeight = this.innerNode.offsetHeight;
+    const viewHeight = this.innerNode.getBoundingClientRect().height;
+    let newViewHeight = viewHeight;
 
     let paddingTop = 0;
     let paddingBottom = 0;
     let showGlyph = false;
 
     // get the content heights of the nodes that are to be shown if have content and cannot be hidden
-    let componentsHeight = this.titleNode.offsetHeight;
+    let componentsHeight = this.titleNode.getBoundingClientRect().height;
     if (this.actionsNode) {
-      componentsHeight += this.actionsNode.offsetHeight;
+      componentsHeight += this.actionsNode.getBoundingClientRect().height;
     }
     if (this.messageNode) {
-      componentsHeight += this.messageNode.offsetHeight;
+      componentsHeight += this.messageNode.getBoundingClientRect().height;
     }
     if (this.dividerNode) {
-      componentsHeight += this.dividerNode.offsetHeight;
+      componentsHeight += this.dividerNode.getBoundingClientRect().height;
     }
 
-    if (this.glyphNode && viewHeight >= componentsHeight + this.glyphNode.offsetHeight && this.innerNode.offsetWidth >= this.glyphNode.offsetWidth) {
+    let totalComponentsHeight = componentsHeight;
+    if (this.glyphNode) {
+      totalComponentsHeight += this.glyphNode.getBoundingClientRect().height;
+      // if inner view node height is lesser than the components along with glyph inside the inner view.
+      // and the height difference is very less (i.e 0.0001) then add that comparision value to the
+      // the inner view height.
+      if (viewHeight < totalComponentsHeight) {
+        if (totalComponentsHeight - viewHeight <= 0.0001) {
```

Comment:
do we need the conditional on line 126? or can we just check ` if (totalComponentsHeight - viewHeight <= 0.0001)`?

Reply:
Removed